database/service_db.go

```diff
@@ -130,8 +191,17 @@ func (e *DecodeError) Error() string {
 	return fmt.Sprintf("Database services: Could not decode service %q", e.ID)
 }
 
-// IsErrNotFound returs true if err is type of ErrNotFound, false otherwise.
+// IsErrNotFound returns true if err is type of ErrNotFound, false otherwise.
 func IsErrNotFound(err error) bool {
 	_, ok := err.(*ErrNotFound)
 	return ok
 }
+
+// ErrSameAlias error returned when there is a service with the same alias.
+type ErrSameAlias struct {
```

Comment:
If 2 services have the same alias the second one will override the first one so for now it's the developers responsibility not to choose 2 services with the same alias. Later with the network and services deployed on the network, the author will be set in the service and if the author doesn't have the right to update the service then the developer will have an error and will need to find a new alias

Reply:
I change the logic to allow an alias to be replaced.
Otherwise, during the development of a service, the developer would have to change the service alias at every deployment, exactly the opposite of what we want to do..